src/test/java/com/wikia/webdriver/common/core/MailFunctions.java

```diff
@@ -19,13 +19,9 @@
 import javax.mail.Session;
 import javax.mail.Store;
 
-/**
- * @author Karol 'kkarolk' Kujawiak
- */
 public class MailFunctions {
 
   private MailFunctions() {
-
   }
 
   public static String getFirstEmailContent(String userName, String password, String subject) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "getFirstEmailContent" is 12 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)